### PR TITLE
New header iphone 5 fix pillars running into the veggie burger

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?
Repeat of: https://github.com/guardian/frontend/pull/14614

`vw` seems to be calculated slightly different in IOS which means that the pillars run into the veggie burger. 

The simplest solution is to just knock down the font-size. 

## What is the value of this and can you measure success?
IOS 7 on an iphone 5 looks better.

## Does this affect other platforms - Amp, Apps, etc?
Nope

## Screenshots
Before:
image

After:
image

## Tested in CODE?
Nope

### CODE REVIEW

**Review by gu-stav — APPROVED** [review]

**Review by zeftilldeath — APPROVED** [review]
Lions
Give
Too
Much

**Comment by paperboyo** [comment]
Extremely handy coincidence I must say! 💅

File: static/src/stylesheets/layout/new-header/_pillar-link.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
     cursor: pointer;
     display: block;
     font-size: 16px;
-    font-size: 5.1vw;
+    font-size: 4.9vw;
```
Comment (sndrs) [inline static/src/stylesheets/layout/new-header/_pillar-link.scss]: classic 👌

Comment (sndrs) [inline static/src/stylesheets/layout/new-header/_pillar-link.scss]: does this mean the font sizes are no longer affected by user zoom?

Comment (gustavpursche) [inline static/src/stylesheets/layout/new-header/_pillar-link.scss]: It is. We decided at some point it would be ok to do only for this breakpoint.

Comment (sndrs) [inline static/src/stylesheets/layout/new-header/_pillar-link.scss]: how come? what if i've zoomed the text system wide?

Comment (gustavpursche) [inline static/src/stylesheets/layout/new-header/_pillar-link.scss]: Thing is there is very very little space in this area and I can't think about any way how to make this way with bigger fonts. Can add it to the list of things to look into ...

Comment (sndrs) [inline static/src/stylesheets/layout/new-header/_pillar-link.scss]: would they need to wrap in that instance, and push the content down?

Comment (gustavpursche) [inline static/src/stylesheets/layout/new-header/_pillar-link.scss]: Yeah, that could be one possible solution. Will add the to the list of things to consider aswell as the animation hint you gave. 👍

**Pull request merged**